nixos/modules/installer/tools/nixos-enter.sh

```diff
@@ -51,11 +55,20 @@ if [[ ! -e $mountPoint/etc/NIXOS ]]; then
     exit 126
```

Comment:
We should probably silent this echo as well.

Reply:
I'm not quite sure whether that's a good idea: if this error occurs, the command from `nixos-enter` isn't even run so silently failing (with an error code) could really confuse people.
I think we're good to print to `stderr` with `--silent` when there's an error in the `nixos-enter` script itself, especially since according to documentation, `--silence` only suppresses activation script output.